README.md

```diff
@@ -1 +1,48 @@
-# vip-inheritance
\ No newline at end of file
+# VIP inheritance annotation
+## GenemapConverter
+A tool to converts the omim 'genemap2.txt' to a tab separated file that can be used by the VEP Inheritance Plugin.
+It creates a line for all gene symbols in the input file.
+###Usage
+```
+usage: java -jar genemap-mapper.jar -i <arg> -c <arg> [-o <arg>] [-f]
+       [-l] [-p] [-d]
+ -i,--input <arg>    Input genemap2.txt location.
+ -o,--output <arg>   Output TSV file
+ -f,--force          Override the output file if it already exists.
+ -d,--debug          Enable debug mode (additional logging).
+
+usage: java -jar genemap-mapper.jar -v
+ -v,--version   Print version.
+```
+
+###Mapping
+|OMIM Inheritance|Annotation|
+|---|---|
+|Y-LINKED|YL|
+|X-LINKED DOMINANT|XD|
+|X-LINKED RECESSIVE|XR|
+|?X-LINKED RECESSIVE|Q_XR|
```

Comment:
I considered, but find it tricky since we're not sure what it means.

for phenotypes the ? means provisional, for inheritance mode it is not documented... probably the same... up for discussion how to treat it.

Reply:
> I considered, but find it tricky since we're not sure what it means.
Shouldn't we find out first before introducing these possibly OMIM-specific inheritance modes?